public/docs/_examples/template-syntax/ts/app/app.component.html

```diff
@@ -351,21 +351,22 @@
 <hr><h2 id="two-way">Two-way Binding</h2>
 <div id="two-way-1">
   <!-- #docregion two-way-1 -->
-  <my-sizer [(size)]="fontSize"></my-sizer>
-  <div [style.font-size.px]="fontSize">Resizable Text</div>
+  <my-sizer [(size)]="fontSizePx"></my-sizer>
```

Comment:
This change seems picayune.  I feel it distracts from the point and starts a conversation that isn't helpful.  I can understand that you feel that being inspecific about the units in the property name provokes a distracting conversation too. I don't actually feel strongly about this so I'd say "whatever, OK" .